signer/core/cliui.go

```diff
@@ -253,12 +256,15 @@ func (ui *CommandlineUI) showAccounts() {
 		fmt.Print("No accounts found\n")
 		return
 	}
-	var out = new(strings.Builder)
+	if len(accounts) < limit {
+		limit = len(accounts)
+		msg = ""
+	}
 	fmt.Fprint(out, "\n------- Available accounts -------\n")
-	for i, account := range accounts {
+	for i, account := range accounts[0:limit] {
```

Comment:
shorter verson:
```suggestion
	for i, account := range accounts[:limit] {
```

Reply:
yea, fair enough - just responding to Felix's question - it would be annoying to have loads and loads of accounts listed. But don't feel super strongly about it.